onnxscript/test/converter_test.py

```diff
@@ -587,6 +597,11 @@ def inner():
                 Temp = op.Abs(X)
                 return op.DummyOp(body=inner)
 
+    def test_attr(self):
+        from onnxscript.test.functions import attr_test
```

Comment:
Sorry one more thing:

Reply:
I prefer to leave it as is. It is a test-suite, but we make it a separate file to keep "onnxscript code" separate (will make it easier to turn off linter/style-checker like tools from being applied to these test-cases). Loading it when needed makes it easier to run test-cases incrementally for debugging etc.